menas/ui/service/RunRestDAO.js

```diff
@@ -15,8 +15,32 @@
 
 class RunRestDAO {
 
-  getSplineUrlTemplate() {
-    return RestClient.getSync(`api/runs/splineUrlTemplate`)
+  constructor(lineageExecutionIdApiTemplate) {
+    this._lineageExecutionIdApiTemplate = lineageExecutionIdApiTemplate;
   }
 
+  get lineageExecutionIdApiTemplate() {
+    return this._lineageExecutionIdApiTemplate
+  }
+
+  getLineageId(outputPath, applicationId) {
+    const urlTemplate = this.lineageExecutionIdApiTemplate;
+    const url = urlTemplate
+      .replace("%s", outputPath)
+      .replace("%s", applicationId);
```

Comment:
Quite the show stopper. Cannot see any Run object. I could see the first one, then I refreshed or something and now I cannot get it to work.

![image](https://user-images.githubusercontent.com/17467768/72592061-06802280-3902-11ea-8f3d-77407e3f292b.png)

Reply:
Is the `menas.lineage.readApiUrl' property set up?
But I will ensure the code doens't go to this branch if it's not. Valid catch 👍

Reply:
Addressed in recent #1140 commit